src/pdf.rs

```diff
@@ -28,8 +28,8 @@ fn initialize_pdfium() {
     )
     .or_else(|_| Pdfium::bind_to_system_library())
     .unwrap();
-    PDFIUM.set(Pdfium::new(bindings)); // Instead of returning the bindings, we
-                                       // cache them in the static initializer
+    let _ = PDFIUM.set(Pdfium::new(bindings)); // Instead of returning the bindings, we
```

Comment:
Perhaps better to do an expect()? Not sure if this can be incorrecly initialised and already set.
```suggestion
    let _ = PDFIUM.set(Pdfium::new(bindings)); // Instead of returning the bindings, we
```